spec/inputs/integration/jms_spec.rb

```diff
@@ -203,17 +260,26 @@
           jms_message
         end
         expect(queue.size).to eql 1
-        expect(queue.first.get('message')).to eql ('hello world')
+        expect(queue.first.get('message')).to eql 'hello world'
         expect(queue.first.get("jms_destination")).to eql(destination)
       end
     end
   end
 end
 
-describe "input/jms", :integration => true do
+describe LogStash::Inputs::Jms, :integration => true do
   let (:message) { "hello World" }
   let (:queue_name) { SecureRandom.hex(8)}
 
+  let (:yaml_section) { 'activemq' }
+  let (:config) {{'yaml_file' => fixture_path("jms.yml"),
+                      'yaml_section' => yaml_section,
+                      'destination' => queue_name,
+                      'pub_sub' => pub_sub,
+                      'interval' => 2}}
+
+  let(:input) { described_class.new(config) }
```

Comment:
since this is the subject of test, I would prefer

```suggestion
  subject(:input) { described_class.new(config) }
```